BUILD_LINUX.md

```diff
@@ -14,60 +14,58 @@ sudo add-apt-repository universe
 sudo apt-get update
 ```
 #### Install build tools:
-1.  First update the repositiories:  
+-  First update the repositiories:  
 ```bash
 sudo apt-get update -y
 sudo apt-get upgrade -y
 ```
-1.  git
+-  git
 ```bash
 sudo apt-get install git -y
 ```
 Verify by git --version  
-1.  g++
+-  g++
 ```bash
 sudo apt-get install g++ -y
 ```
 Verify by g++ --version  
-1.  *Ubuntu 18.04* cmake
+-  *Ubuntu 18.04* cmake
 ```bash
 sudo apt-get install cmake -y
 ```
 Verify by cmake --version  
-1. *Ubuntu 16.04* cmake  
+- *Ubuntu 16.04* cmake  
 ```bash
 wget https://cmake.org/files/v3.14/cmake-3.14.2-Linux-x86_64.sh
 sudo sh cmake-3.14.2-Linux-x86_64.sh --prefix=/usr/local --exclude-subdir
 ```
 #### Install build dependencies:
-1.  OpenSSL  
+-  OpenSSL  
 ```bash
 sudo apt-get install libssl-dev
 ```
 Verify with `openssl version`  
-1.  OpenGL  
-Verify (first install mesa-utils - `sudo apt install mesa-utils -y`) by `glxinfo | grep "OpenGL version"`  
+- OpenGL
 ```bash
 sudo apt-get install libgl1-mesa-dev -y
 sudo ln -s /usr/lib/x86_64-linux-gnu/libGL.so.346.35 /usr/lib/x86_64-linux-gnu/libGL.so.1.2.0
 ```
+- Verify OpenGL:
+  - First install mesa-utils with the command `sudo apt install mesa-utils -y`
+  - Then run `glxinfo | grep "OpenGL version"`  
 #### To compile interface in a server you must install:
 ```bash
 sudo apt-get -y install libpulse0 libnss3 libnspr4 libfontconfig1 libxcursor1 libxcomposite1 libxtst6 libxslt1.1
 ```
-1.  Misc dependencies
+-  Misc dependencies
 ```bash
 sudo apt-get install libasound2 libxmu-dev libxi-dev freeglut3-dev libasound2-dev libjack0 libjack-dev libxrandr-dev libudev-dev libssl-dev zlib1g-dev
 ```
-1.  To compile interface in a server you must install:
+-  Install Python 3 and required packages
```

Comment:
":" missing from end. Similarly for other instructions, above.

Reply:
Fixed